scripts/fgi/kube.py

```diff
@@ -310,13 +307,12 @@ def helm_s3_init(workspace):
     """Initializes the S3 bucket used as an internal Helm repo for Forge"""
     bucket_url = WORKSPACE_CHART_BUCKETS[workspace]
     subprocess.run(
-        f"helm plugin install https://github.com/C123R/helm-blob.git || true",
-        shell=True,
-        check=True
+        ["helm", "plugin", "install", "https://github.com/C123R/helm-blob.git"],
+        check=False
     )
     subprocess.run(
         ["helm", "blob", "init", f"s3://{bucket_url}/charts"],
-        check=True
+        check=False
```

Comment:
init fails if the repo had already been initialized. it's dirty but we could check the output I guess. if it doesn't init properly though, it will fail at a later step about auth or something